Tests/BugsnagPerformanceTests/ViewControllerSpanTests.mm

```diff
@@ -90,14 +90,14 @@ - (void)testAutoViewControllerDidAppearWillDisappear {
     UIViewController *controller = [MyTestViewController new];
     [controller loadView];
     [controller viewDidLoad];
-    XCTAssertEqual(0U, perf->testing_getBatchCount());
+    XCTAssertEqual(2U, perf->testing_getBatchCount());
     [controller viewDidAppear:controller];
-    XCTAssertEqual(1U, perf->testing_getBatchCount());
+    XCTAssertEqual(4U, perf->testing_getBatchCount());
 
     controller = [MyTestViewController new];
     [controller loadView];
     [controller viewDidLoad];
-    XCTAssertEqual(1U, perf->testing_getBatchCount());
+    XCTAssertEqual(6U, perf->testing_getBatchCount());
 // Temporarily disabled to stop crashes while we build a more complete solution
```

Comment:
Oh could you uncomment these 2 disabled lines again? I forgot to do it after the refactor.